Userland/Shell/Parser.cpp

```diff
@@ -5,7 +5,9 @@
  */
 
 #include "Parser.h"
+#include "Kernel/API/POSIX/unistd.h"
```

Comment:
:thonk: feels like a clangd driven include, <unistd.h> is already included

Reply:
clangd moment